Source/Response.swift

```diff
@@ -101,6 +107,7 @@ extension DataResponse: CustomStringConvertible, CustomDebugStringConvertible {
 
         return """
         [Request]: \(requestDescription)
+        [Parameters]: \(parameters)
         [Response]: \n\(responseDescription)
         [Data]: \(data?.description ?? "None")
```

Comment:
Since you're adding the "Request Body", you should do the same thing here: generate a `String` from the response `Data` and update this field to be `[Response Body]`.

Reply:
I have added [Response Body]